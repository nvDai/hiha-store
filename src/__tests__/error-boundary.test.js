import React from 'react'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import ErrorBoundary from '../components/error-boundary/error-boundary.component'

describe('ErrorBoundary', () => {
	it('should render without crashing', () => {
		// const state = {
		// 	hasError: false,
		// }
		// const wrapper = shallow(<ErrorBoundary />)
		// console.log(wrapper.instance().getDerivedStateFromError())
		// expect(toJson(wrapper)).toMatchSnapshot()
	})
})
