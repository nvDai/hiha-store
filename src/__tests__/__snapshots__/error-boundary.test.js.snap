// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary should render without crashing 1`] = `""`;
